### ISSUE TITLE
[Suggestion] Select capitans

### ISSUE THREAD

**Issue by GrzesiuS** [shobhit-pathak/MatchZy#44]
Hello,

It would be nice to have an option to select two captains of the teams and they select the rest of the teammates before starting the match.

A similar option I had in https://github.com/splewis/csgo-pug-setup

**Comment by shobhit-pathak** [comment]
Hey!

Yes this is a nice suggestion and option to have, but currently I'm working towards setting up of matches using JSON (similar to get5) so implementing this captain system may take some time 😅

Thanks!

**Comment by GrzesiuS** [comment]
Are you looking for additional help with work? If you introduce me into project I can spend some time on it.
